api-inference-community/tests/test_dockers.py

```diff
@@ -240,6 +240,10 @@ def test_keras(self):
             "keras", "image-classification", "nateraw/keras-mnist-convnet"
         )
 
+    def test_doctr(self):
```

Comment:
Please also add a GH Actions workflow to test this image. You can just duplicate this and do some minor modifications https://github.com/huggingface/huggingface_hub/blob/main/.github/workflows/python-api-stanza.yaml